handbook/engineering/README.md

```diff
@@ -116,7 +116,7 @@ To provide full-time focus to the role, the on-call engineer is not expected to
 
 ### Create a release candidate
 
-All minor releases go through the release candidate process before they are published. A release candidate for the next minor release is created on the Tuesday before the release at 11:00 AM Pacific. A release candidate branch is created at `minor-fleet-v4.x.x` and no additional feature work is merged without EM and QA approval.
+All minor releases go through the release candidate process before they are published. A release candidate for the next minor release is created on the first Monday of each sprint at 9:00 AM Pacific. A release candidate branch is created at `minor-fleet-v4.x.x` and no additional feature work is merged without EM and QA approval.
```

Comment:
And the expectation would be to release by ~ the first thursday of the next sprint?

Reply:
Thoughts on moving this back a bit to around the time of sprint demos? So 9a PT the last Friday of the sprint? Or maybe earlier/significantly later that day to give time to actually perform the ritual?

Otherwise we're going to have two problems off the top of my head:

1. Sprint demos will start getting, erm, optimistic about what we're shipping because there will be items that _could_ make the cut that haven't merged yet.
2. For those items, making the cut will require working in bits and pieces of Friday afternoon and/or over the weekend, so the weekend prior to the start of a sprint will wind up being the highest-pressure few days of our entire release workflow. Which...seems like a pretty direct contributor to burnout, versus having that weekend free by default.

Reply:
Also, given the current QA backlog, making this switch now would mean a release with ~3.75 weeks of dev (because work done after the previous release cut is already in the current release, _plus_ we have the bunch of work going into the next release). Which means that doing this change now makes the QA backlog worse, which cascades catching unreleased bugs into the next dev sprint, which decreases the efficacy of capacity planning for that sprint.

This feels like a change that'd be better to do when we tag a minor release the Monday or Tuesday after sprint demos, but not before then.

Reply:
^ That makes sense to me. Thanks for pointing those out @iansltx

Reply:
@iansltx Great points! I've changed to the last Friday at 8:00 AM Pacific.